apps/s_client.c

```diff
@@ -2172,6 +2172,16 @@ int s_client_main(int argc, char **argv)
              * HTTP/d.d ddd Reason text\r\n
              */
             mbuf_len = BIO_gets(fbio, mbuf, BUFSIZZ);
+            if (mbuf_len < 12) {
+                /* '12' stands for the length of 'HTTP/d.d ddd' */
```

Comment:
Errr, it would be slower to calculate the length of that string every time than just put a constant value there.